packages/components/tests/unit/helpers/validateCssAndMatchSnapshot.ts

```diff
@@ -2,16 +2,24 @@ import { expect } from '@jest/globals';
 
 /*
   TODO: Existing tests for:
-    - !important on host display style
-    - !important on slotted styles
     - @hover media query for hover styles
-    - Host hidden styles
     should also be covered for all getComponentCss conditions
 */
-export const validateCssAndMatchSnapshot = (css: string) => {
-  // We shouldn't use visibility: visible since it cannot be overwritten, use inherit instead
+export const validateCssAndMatchSnapshot = (css: string, isInternal = false) => {
```

Comment:
Could remove the `isInternal` parameter and extract the name of the test suite and/or file tested via `expect.getState()` to get the component name and then retrieve the `isInternal` information via `componentMeta`. This way there is no maintenance and no checking needed what the parameter does.